css/theme-weebly.scss

```diff
@@ -931,4 +931,72 @@ div.Notice {
 	margin-right: 4px;
 }
 
+// Popover
```

Comment:
Can you move this CSS out of the Orbit Overrides section. I'd create a new Popover section below Cards.
`/* POPOVER ------------------------------------------------------------------- */`